src/utils/command.cpp

```diff
@@ -18,7 +19,95 @@
 
 POLYBAR_NS
 
-command::command(const logger& logger, string cmd) : m_log(logger), m_cmd(move(cmd)) {
+command<output_policy::IGNORED>::command(const logger& logger, string cmd) : m_log(logger), m_cmd(move(cmd)) {}
+
+command<output_policy::IGNORED>::~command() {
+  if (is_running()) {
+    terminate();
+  }
+}
+
+/**
+ * Execute the command
+ */
+int command<output_policy::IGNORED>::exec(bool wait_for_completion) {
+  if ((m_forkpid = fork()) == -1) {
+    throw system_error("Failed to fork process");
+  }
+
+  if (process_util::in_forked_process(m_forkpid)) {
+    setpgid(m_forkpid, 0);
+    process_util::exec_sh(m_cmd.c_str());
```

Comment:
Here, the forked process inherits polybar's stdin, stdout, and stderr. So any output will become part of polybar's output. Is this intended?

For one, it makes sense that people can see the output of their script for debugging, but the polybar output may get spammed with command outputs. I'm not sure what the best approach is here.

Reply:
I think this is the only point that still needs to be addressed.
I have thought about this a bit and I believe processes spawned like this should have their stdout and stderr redirected to `/dev/null`. If they don't do that, they will forever output in the shell running polybar (even after polybar is closed). This is especially annoying for long-running programs.

I think voiding the output like this is enough:

```c
int fd_null = open("/dev/null", O_WRONLY);
dup2(fd_null, STDOUT_FILENO);
close(fd_null);
```

And the same for stderr (with error checks of course).